boot/src/test/resources/application.properties

```diff
@@ -0,0 +1 @@
+api.token.dd=
```

Comment:
Do we need it here?

Reply:
Yes. The tests won`t run. I tried different solutions, but only this one works